data-model/SDP-sample-records/ms-object_sample1.json

```diff
@@ -0,0 +1,85 @@
+{
+  "id": "ark:/21198/z1x07bdf",
+  "type": "Manuscript Object",
```

Comment:
let's make the types more machine-friendly - "ms_object" "part" and "text_art" or something like that

Reply:
Also review for some of the other controlled vocabulary fields (e.g., 'form' that has "Codex", "Quire(s)", etc.). We should have a short code which will eventually be part of the term list